invenio_rdm_records/services/review/service.py

```diff
@@ -180,3 +173,42 @@ def submit(self, identity, id_, data=None, revision_id=None, uow=None):
         uow.register(RecordIndexOp(draft, indexer=self.indexer))
 
         return request_item
+
+    @unit_of_work()
+    def submit(self, identity, id_, data=None, revision_id=None, uow=None):
```

Comment:
Why splitting these 2 methods? In my mind there should be only one, no option: if I can direct publish, it should just happen, and not by choosing another endpoint?